### ISSUE TITLE
Add binaries to releases section

### ISSUE THREAD

**Issue by dene14** [AdaptiveConsulting/nexus-casc-plugin#10]
There is onlt source code tarballs in releases section on github right now.
Can we add binaries (*.kar) in addition or as a replacement for it?

**Comment by bdellegrazie** [comment]
Hi @dene14 - I'll need to check internally where we can host them. I don't think it will be a huge problem but I haven't thus far because of that.

Just FYI, it is trivially simple to build (JDK 8 + `mvn package` will get you the .kar file) - i've updated the documentation to include a building section.

**Comment by dene14** [comment]
Yeah, I know... I just noticed that you have pipeline anyway, just pushing to your private artifactory.. I believe Github Releases can be used for that... For e.g.: https://github.com/roboll/helmfile/releases

**Comment by csullivannet** [comment]
Since this project is open source, can it use GitHub Actions for free? I might not be able to get to it for a few weeks, but I could look at using Actions to create and publish the `.kar` archive.

**Comment by csullivannet** [comment]
@dene14 @bdellegrazie I've opened a PR that automatically builds and publishes, it generaties artifacts for regular commits and publishes to the release:
https://github.com/AdaptiveConsulting/nexus-casc-plugin/pull/12

**Comment by bdellegrazie** [comment]
@csullivannet I have merged your changed but at present it doesn't look like its producing the artefacts or I haven't tied it in properly. I'll recheck it over the weekend.

**Comment by csullivannet** [comment]
I can see that you created a release from the commit after you merged the workflow. It should have then kicked off the workflow to create the release bundle.

Let me know how your testing over the weekend goes.

**Comment by FOTempel** [comment]
Our deployment tooling for Nexus pulls its plugins from https://repo1.maven.org/maven2/org/sonatype/nexus/plugins/, that would be a great place to put it. Having to compile a plugin ourselves would introduce chicken and egg problems as built artifacts belong in an artifact manager which we're trying to deploy, which we can't because there's no CasC (yet)...
This can be worked around, sure, by including built binaries in the deployment code, but that's ugly and will introduce new problems.
